### DISCUSSION TITLE
Commercial license option

### DISCUSSION THREAD

**Discussion by alsmirn** [dcm4che/dcm4che#1132] [General]
Thanks for your awesome work!

Do you by any chance offer alternative commercial license as well? 
If yes, what are the conditions for it?

**Comment by gunterze** [comment]
No. You already have the choice between use under the terms of
- the Mozilla Public License Version 1.1, or
- the GNU General Public License Version 2 or later (the "GPL"), or
- the GNU Lesser General Public License Version 2.1 or later (the "LGPL")

So there is only the restriction, that changes on individual source files shall been published as open source again. The source of commercial products built on or including dcm4che artifacts does not need to be published as open source.

**Reply by alsmirn** [reply]
I understand that's my question sounds a bit strange, but we have such atypical case. Thank you for the public answer, that's good enough for us.
